src/marshmallow/__init__.py

```diff
@@ -11,10 +11,10 @@
 )
 from marshmallow.utils import EXCLUDE, INCLUDE, RAISE, pprint, missing
 from marshmallow.exceptions import ValidationError
-from distutils.version import LooseVersion
+from packaging.version import Version
 
 __version__ = "3.14.0"
-__version_info__ = tuple(LooseVersion(__version__).version)
+__version_info__ = Version(__version__).release
```

Comment:
This is missing the prerelease information that the previous version info contained.

```py
>>> tuple(LooseVersion('3.0.0b16').version)
(3, 0, 0, 'b', 16)
>>> Version('3.0.0b16').release
(3, 0, 0)
>>> v = Version('3.0.0b16')
>>> v.release + v.pre
(3, 0, 0, 'b', 16)
```

We could avoid the dependency by parsing the version string ourself.

```py
parts = filter(None, re.split(r'\.|(\D+)', __version__))
__version_info__ = tuple(int(p) if p.isdigit() else p for p in parts)
```

Reply:
I wouldn't be willing to go that route personally and would ask someone else to take this on in that case. I think it's bad practice to purposefully avoid dependencies to avoid the recommended migration path. 

Your call which way you want to go. I'm fine closing this as a wontfix in this state.

Reply:
Let me know if you want all the segments represented from the version regex in it's order sequence:

```
VERSION_PATTERN = r"""
    v?
    (?:
        (?:(?P<epoch>[0-9]+)!)?                           # epoch
        (?P<release>[0-9]+(?:\.[0-9]+)*)                  # release segment
        (?P<pre>                                          # pre-release
            [-_\.]?
            (?P<pre_l>(a|b|c|rc|alpha|beta|pre|preview))
            [-_\.]?
            (?P<pre_n>[0-9]+)?
        )?
        (?P<post>                                         # post release
            (?:-(?P<post_n1>[0-9]+))
            |
            (?:
                [-_\.]?
                (?P<post_l>post|rev|r)
                [-_\.]?
                (?P<post_n2>[0-9]+)?
            )
        )?
        (?P<dev>                                          # dev release
            [-_\.]?
            (?P<dev_l>dev)
            [-_\.]?
            (?P<dev_n>[0-9]+)?
        )?
    )
    (?:\+(?P<local>[a-z0-9]+(?:[-_\.][a-z0-9]+)*))?       # local version
"""
```